addons/policies/bootstrap_entity_policies.json

```diff
@@ -3108,6 +3108,190 @@
                     "entity-read"
                 ]
             }
+        },
+
+        {
+            "typeName": "AuthPolicy",
+            "attributes": {
+                "name": "CUD_BUSINESS_POLICY",
+                "qualifiedName": "CUD_BUSINESS_POLICY",
+                "policyCategory": "bootstrap",
```

Comment:
Just for confirmation, are there any relationships defined between any of these new types, if yes i think we need to bootstrap relationship policies as well